### PR TITLE
 Simplify travis.yml (openjdk10 and openjdk11)

### PR DESCRIPTION
**Pull request by epeee**
... and remove no longer needed wget of install-jdk.sh

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -41,9 +41,6 @@ branches:
   except:
   - /^v\d/
 
-before_install:
-  - wget https://raw.githubusercontent.com/sormuras/bach/master/install-jdk.sh
```
Comment (TimvdLippe) [inline .travis.yml]: Do we maybe need this for future jdk versions?

Comment (szpak) [reply .travis.yml]: We can always bring it back.

Comment (szpak) [reply .travis.yml]: Travis under the hood uses that script (in some version).

**Comment by epeee** [comment]
@TimvdLippe we should pass on java11 once #1558 is merged.

File: .travis.yml (hunk 1 above)
Comment (epeee) [reply .travis.yml]: We can add openjdk-ea in order to use java12 early access builds.

Comment (szpak) [reply .travis.yml]: Good idea with Java 12 EA

**Comment by szpak** [comment]
I merged #1558. @TimvdLippe Feel free to merge it once you are convinced :).

**Comment by epeee** [comment]
@TimvdLippe I just rebased and got rid of `allow_failures`.

**Pull request merged**